### PR TITLE
replace labels

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: src/form/Attributes/Taxonomy/hooks/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export const useTaxonItemLabelExtractor = nodeDef => {
+  const language = useSelector(surveySelectors.getSelectedSurveyLanguage);
+  const defaultVisibleFields = useSelector(
+    appSelectors.getSettingsPreferencesSurveyTaxonomiesDefaultVisibleFields,
+  );
+
+  const _labelStractor = useCallback(
+    item => {
+      console.log(
```
Comment (SteRiccio) [inline src/form/Attributes/Taxonomy/hooks/index.js]: you should remove this console.log

File: src/form/Attributes/Taxonomy/hooks/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        NodeDefs?.getVisibleFields?.(nodeDef) || defaultVisibleFields,
```
Comment (SteRiccio) [inline src/form/Attributes/Taxonomy/hooks/index.js]: can NodeDefs be null or undefined?! or can getVisibleFields be undefined?!

**Review by SteRiccio — APPROVED** [review]
approved with one comment (the console.log)

File: src/form/Attributes/Taxonomy/hooks/index.js (hunk 2 above)
Comment (ramirobg94) [reply src/form/Attributes/Taxonomy/hooks/index.js]: I tried before the merge in arena-core this is why I have errors and I forget to remove this console log, thanks

**Pull request merged**